CapstoneProject/Assets/Scripts/Core/GameState/GameState.cs

```diff
@@ -14,6 +14,57 @@ public enum States
             UI
         }
 
+
+        public short recentDeath;
+        public short relationshipStatus;
+
+        public short affiliation;
+        public short runsCompleted;
+        public short playerProgress;
+        public short NPCQuestProgress;
+        public string currentNPC;
+        public short dialogueStage;
+
+        public short ReturnRecentDeaths()
+        {
+            return recentDeath;
+        }
+
+        public short ReturnrelationshipStatus()
+        {
+            return relationshipStatus;
+        }
+        
+        public short ReturnAffiliation()
+        {
+            return affiliation;
+        }
+
+        public short ReturnRunsCompleted()
+        {
+            return runsCompleted;
+        }
+
+        public short ReturnPlayerProgress()
+        {
+            return playerProgress;
+        }
+
+        public short ReturnNPCQuestProgress()
+        {
+            return NPCQuestProgress;
+        }
+
+        public string ReturnCurrentNPC()
+        {
+            return currentNPC;
+        }
+
+        public short ReturnDialogueStage()
+        {
+            return dialogueStage;
+        }
```

Comment:
these return X,Y,Z functions are odd for function name. Why not just use properties?
https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-properties